internal/wire/header.go

```diff
@@ -33,6 +33,7 @@ type Header struct {
 	Type         protocol.PacketType
 	IsLongHeader bool
 	KeyPhase     int
+	PayloadLen   protocol.ByteCount
```

Comment:
Maybe it's worth splitting the long header and short header? Or have a substruct with only the long header / short header specific fields?